### PR TITLE
Rawgit Support

### PR DESCRIPTION
**Pull request by GordonSmith**
Allow demos to work with rawgit.
Convert paths to point to files and not folders.

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@buunguyen Please Review.

Note:  I have given up trying to workaround the c3 name clashes between src/c3 and bower_components/c3 and have renamed it to src/c3chart (wish I had done it last week now).

http://rawgit.com/GordonSmith/Visualization/ENABLE_RAWGIT/demos/test.html
http://rawgit.com/GordonSmith/Visualization/ENABLE_RAWGIT/demos/dermatology.html

**Comment by buunguyen** [comment]
LGTM. 

The name clash is pain, I spent a lot of time working around it in my past commits. I reviewed a commit from @mlzummo last week, there's also a name clash with Amcharts. Probably we should avoid all name clashes from now on, would make the config cleaner and avoid trouble.

File: demos/config.js [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
     baseUrl: ".",
     paths: {
-        'async': '../bower_components/requirejs-plugins/src/async',
-        'css': '../bower_components/require-css/css',
-        'text': '../bower_components/text/text',
-        'goog': '../bower_components/requirejs-plugins/src/goog',
-        'propertyParser': '../bower_components/requirejs-plugins/src/propertyParser',
-
-        "lodash": "./bower_components/lodash-compat",
+        "css": ["http://rawgit.com/guybedford/require-css/0.1.8/css.min", "../bower_components/require-css/css"],
```
Comment (mlzummo) [inline demos/config.js]: i wouldn't use rawgit as a CDN

**Comment by mlzummo** [comment]
looks good, i see that this is in demos anyways so the pro vs dev cdn thing doesnt really apply

**Pull request merged**